insights/parsers/openshift_get_with_config.py

```diff
@@ -14,44 +14,93 @@
+        asb-access                                                            /asb-access                                                                                                                                            openshift-ansible-service-broker/asb-client
+        asb-auth                                                              /asb-auth                                                                                                                                              openshift-ansible-service-broker/asb
+        auth-delegator-openshift-template-service-broker                      /system:auth-delegator                                                                                                                                 openshift-template-service-broker/apiserver
+        basic-users                                                           /basic-user                                                                                             system:authenticated
+        cluster-admin                                                         /cluster-admin                                                                                          system:masters
+        cluster-admin-0                                                       /cluster-admin                                                                                                                                         insights-scan/insights-scan
+
+    Examples:
+        >>> type(oc_get_clusterrolebinding_with_config)
+        <class 'insights.parsers.openshift_get_with_config.OcGetClusterRoleBindingWithConfig'>
+        >>> oc_get_clusterrolebinding_with_config.rolebinding["admin"]
+        '/admin'
+    """
 
     def parse_content(self, content):
-        self.rolebinding = {}
         if len(content) < 2 or "NAME" not in content[0]:
-            raise ParseException("{0}: invalid content".format(self.__class__.__name__))
+            raise ParseException("invalid content" if content else 'empty file')
+
+        self.data = parse_fixed_table(content)
+        self.rolebinding = {}
+
+        for rolebinding in self.data:
+            rolebinding_name = rolebinding["NAME"].strip()
+            rolebinding_role = rolebinding["ROLE"].strip()
+            if rolebinding_name and rolebinding_role:
+                self.rolebinding[rolebinding_name] = rolebinding_role
+            else:
+                raise ParseException("invalid ROLE format" if content else 'empty file')
 
-        for line in content[1:]:
-            item_list = line.split()
-            if '/' not in line or len(item_list) < 2:
-                raise ParseException("{0}: invalid ROLE format".format(self.__class__.__name__))
-            self.rolebinding[item_list[0]] = item_list[1]
+    def __contains__(self, item):
```

Comment:
Rename the `self.rolebinding` to `self.data` and mix this parser with `LegaryItemAccess`